src/pages/_app.js

```diff
@@ -7,28 +7,24 @@ import Alert from "@/src/components/Alert";
 import useLocalStorageState from "use-local-storage-state";
 
 export default function App({ Component, pageProps }) {
-  const [transactions, setTransactions] = useLocalStorageState("transactions", {
-    defaultValue: transactionsData,
-  });
-  const [mode, setMode] = useState("default");
   const router = useRouter();
-  const [alert, setAlert] = useState({
-    isOpen: false,
-    alertMessage: "",
-  });
+  const [transactions, setTransactions] = useLocalStorageState("transactions", { defaultValue: transactionsData });
+  const [alert, setAlert] = useState({ isOpen: false, alertMessage: "" });
+  console.log(transactions);
 
+  //-------------Alert logic-------------
   function handleAlert(alertMessage) {
-    setAlert({ isOpen: true, alertMessage});
+    setAlert({ isOpen: true, alertMessage });
```

Comment:
Is this working as expected? I might be wrong but I thought you still need the key and write ```alertMessage: alertMessage``` to make it dynamic.

Reply:
Nope does work as expected.